src/main/java/org/isf/accounting/gui/PatientBillEdit.java

```diff
@@ -386,7 +401,14 @@ private void initCurrencyCod() {
 	}
 
 	private void setBill(Bill bill) {
-		this.thisBill = bill;
+		try {
+			this.thisBill = (Bill) bill.clone();
+			this.thisBill.setAdmission(bill.getAdmission());
+			this.thisBill.setBillPatient(bill.getBillPatient());
+			this.thisBill.setPriceList(bill.getPriceList());
+		} catch (CloneNotSupportedException cnse) {
+			cnse.printStackTrace();
```

Comment:
Why isn't this Logged instead of printing a stack trace?

Reply:
It is a very rare case, I think it can be tracked for more info

Reply:
you could add the full stack trace with all info to the log as well if that is the case, no sure if keeping the printStackTrace could add advantages for GUI debugging